hdr_schemata/definitions/HDRUK/MeasuredProperty.py

```diff
@@ -1,6 +1,6 @@
-from pydantic import RootModel
+from pydantic import RootModel, constr
 
 class MeasuredProperty(RootModel):
-    pass
+    root: constr(min_length=1, max_length=100)
```

Comment:
I added this so that form hydration had something to validate for this field but is this a breaking change to the GWDM? I wouldn't want to introduce one of those.

Reply:
I updated the `MeasuredProperty` definition just for Hdruk221 so this change should no longer affect the GWDM or earlier HDR schemas.